src/style.css

```diff
@@ -1,3 +1,74 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
 body {
+  font-family: "Roboto", serif;
   margin: 0;
+  font-size: 12px;
+  color: #000;
+  background: #fff;
+}
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  border: none;
+  box-sizing: border-box;
+}
```

Comment:
resetting styles with * is a bad practice, resetting styles with `*` is a bad practice, it is better to work on each element separately to get the expected behavior